pyproject.toml

```diff
@@ -66,7 +66,8 @@ notebook = [
 testpaths = ["tests"]
 filterwarnings = [
     "error",
-    "ignore:'write_like_original':DeprecationWarning:pydicom:"
+    "ignore:'write_like_original':DeprecationWarning:pydicom:",
+    "ignore:Anomaly Detection has been enabled:UserWarning", #torch.autograd
```

Comment:
I am a bit puzzled here because I thought that something like 
`ignore:Anomaly Detection has been enabled:UserWarning:torch`
or
`ignore:Anomaly Detection has been enabled:UserWarning:torch.*`
should work, but I could not get it to work. If anybody knows how to get it to work with specifying the module correctly I would be very happy to know.